desktop/kanshi/kanshi.SlackBuild

```diff
@@ -66,9 +67,9 @@ set -e
 rm -rf $PKG
 mkdir -p $TMP $PKG $OUTPUT
 cd $TMP
-rm -rf $PRGNAM-$VERSION
-tar xvf $CWD/$PRGNAM-$VERSION.tar.gz || tar xvf $CWD/$VERSION.tar.gz
```

Comment:
basically restore and adapt this line which handles both naming options

Reply:
Thx - updated via commit https://github.com/SlackBuildsOrg/slackbuilds/pull/9272/commits/81a976f00e34ba4373b2b564160b6f15f37f0315 which handles this via short commit hashes